string key    # Key of the requested parameter
---
bool result   # True, if a parameter with the requested key exists
